Source/Elasticurl/CommandLine.swift

```diff
@@ -4,8 +4,8 @@
 import Foundation
 import AwsCCommon
 // swiftlint:disable trailing_whitespace
-// Todo: remove/refactor
-public struct CommandLineParser {
+
+struct CommandLineParser {
```

Comment:
I have moved this class to ElasticCurl and made it private for now. We can nuke it in the future and used a built in command line parser.